src/internal/sdata/sql.go

```diff
@@ -38,9 +38,7 @@ func (m *SQLTupleWriter) Flush() error {
 	// flatten list of Tuple
 	var values Tuple
 	for r := range m.buf {
-		for c := range m.buf[r] {
-			values = append(values, m.buf[r][c])
-		}
+		values = append(values, m.buf[r]...)
```

Comment:
Lol this seems unrelated, but good to merge nonetheless.

Reply:
Oh, the linter demands it now.